test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -85,7 +82,9 @@ async function openDappAndSwitchChain(driver, dappUrl, chainId) {
       await driver.findClickableElement(
         '[data-testid="page-container-footer-next"]',
       );
-      await driver.clickElement('[data-testid="page-container-footer-next"]');
+      await driver.clickElementAndWaitForWindowToClose(
```

Comment:
fixes race condition. We need to wait until the dialog is closed before proceeding